tox.ini

```diff
@@ -18,7 +18,7 @@ deps =
 # tox -e py35 -- project/tests/test_file.py::TestClassName::test_method
 commands =
     python util.py clean-tests
-    pytest -vv --cov {posargs}
+    pytest -vv -rs --cov="" {posargs}

```

Comment:
@alanmcruickshank when I tried running only one file with the current tox I couldn't:

```bash
tox -e py38 -- test/core/rules/rule_test_cases_test.py::test__rule_test_case
```

passing `""` to `--cov` fixed it

the `-rs` is to display "Skipped" messaged

Reply:
I found this too and resorted to just calling pytest directly. Thanks!

Reply:
I've just done some testing, this this current change breaks test coverage.

The solution which achieves both aims is:
```
pytest -vv -rs --cov=sqlfluff {posargs}
```

...or after merging in master it should probably be.
```
pytest -vv -rs --cov=sqlfluff {posargs:-m "not dbt"}
```